homeassistant/components/sensor/gtfs.py

```diff
@@ -219,11 +223,16 @@ def icon(self):
         """Icon to use in the frontend, if any."""
         return ICON
 
+    @property
```

Comment:
That being said, we can also just remove it. Maybe not so important to have it in the attributes?